stack.yaml

```diff
@@ -2,9 +2,17 @@ resolver: nightly-2017-06-19
 extra-package-dbs: []
 extra-deps:
 - 'optparse-applicative-0.14.0.0'
-- 'pandoc-citeproc-0.10.5.1'
+- 'pandoc-citeproc-0.12.1'
 - 'QuickCheck-2.10.0.1'
 - 'tasty-quickcheck-0.9'
+- 'pandoc-2.0.2'
+- 'cmark-gfm-0.1.3'
+- 'doctemplates-0.2.1'
+- 'hslua-0.9.2'
+- 'skylighting-0.4.3.2'
+- 'texmath-0.10'
+- 'pandoc-types-1.17.3'
+- 'hs-bibutils-6.2.0.1'
```

Comment:
If you bump to the latest nightly, these should now be in the index.

Reply:
Updated to `nightly-2017-11-19`.